kinds/dashboard/dashboard_kind.cue

```diff
@@ -33,8 +33,9 @@ lineage: seqs: [
 				// Timezone of dashboard,
 				timezone?: *"browser" | "utc" | "" @grafanamaturity(NeedsExpertReview)
 				// Whether a dashboard is editable or not.
-				editable:     bool | *true
-				graphTooltip: #DashboardCursorSync @grafanamaturity(NeedsExpertReview)
+				editable: bool | *true
+				// Configuration of dashboard cursor sync behavior.
+				graphTooltip: #DashboardCursorSync | *0
```

Comment:
@sdboyer  - should I define the default here if it's already provided by `DashboardCursorSync`?

Reply:
Having it default on `DashboardCursorSync` is enough in this case.